InnerEye/ML/lightning_metrics.py

```diff
@@ -248,17 +249,17 @@ def compute(self) -> torch.Tensor:
         return auc(recall, prec)
 
 
-class BinaryCrossEntropy(ScalarMetricsBase):
+class BinaryCrossEntropyWithLogits(ScalarMetricsBase):
     """
     Computes the cross entropy for binary classification.
```

Comment:
Maybe add in docu here that this is a metric that must be computed off the logits.